Terminal Commands.md

```diff
@@ -1,6 +1,12 @@
 Some commands can be found in the `bin` directory.
 
-`ls` - list files
+`ls` - list files<br>
+`ls -a` - display hidden files and folders<br>
+`ls -R` - provide a tree listing of directory structure.<br>
+`ls -s` - list sorted according to size<br>
```

Comment:
Remove the 3 sorting vaale flags...not used very often. We should not spoonfeed a lot of flags. People should be motivated to use `man` command.